app/scripts/controllers/membership.js

```diff
@@ -88,16 +91,16 @@ angular
           .then(function() {
             resetForm();
             refreshRoleBindingList();
-            showAlert('rolebindingCreate', 'success', messages.update.subject.success({
+            showAlert('rolebindingCreate', 'success', escaped.update.subject.success({
               roleName: role.metadata.name,
               subjectName: _.escape(newSubject.name)
```

Comment:
Ah, true, removing the `_.escape(` on these as redundant.
Though, even just with the template escape sequence `<<<bar` still appears as:

![screen shot 2016-10-25 at 12 09 53 pm](https://cloud.githubusercontent.com/assets/280512/19694114/08a9781a-9aac-11e6-862a-c4b0c4152b07.png)